### PR TITLE
Updating CONTRIBUTING.md with slack url

### LINKED ISSUE #1655: Error while connecting to kanister.io slack
**Describe the bug**

**To Reproduce**
1. Go to https://github.com/kanisterio/kanister/blob/master/CONTRIBUTING.md
2. Click on Slack hyperlink
3. It will redirect to https://github.com/kanisterio/kanister/blob/master/kanisterio.slack.com and show error message as shown in attached screenshot

**Expected behavior**
URL should redirect to slack signup/login

**Screenshots**
Screenshot 2022-10-01 at 12 12 00 PM

### PR DESCRIPTION
**Pull request by Sagar2366**
## Change Overview
Updating Slack URL in CONTRIBUTING.md

## Pull request type

Please check the type of change your PR introduces:
- [x] :world_map: Documentation

## Issues 

- fixes #1655

### CODE REVIEW

**Review by PrasadG193 — APPROVED** [review]
Thanks for fixing this @Sagar2366

**Comment by PrasadG193** [comment]
@ihcsim not related, but should we have Slack invite link here instead of the Workspace login link?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you are interested in contributing, start by reading this document. Please
 also take a look at our [code of conduct](CODE_OF_CONDUCT.md).
 
 If you have any questions at all, do not hesitate to reach out to us on
-[Slack](kanisterio.slack.com).
+[Slack](https://kanisterio.slack.com).
```
Comment (ihcsim) [inline CONTRIBUTING.md:11]: Can we use the slack invite link instead? https://join.slack.com/t/kanisterio/shared_invite/enQtNzg2MDc4NzA0ODY4LTU1NDU2NDZhYjk3YmE5MWNlZWMwYzk1NjNjOGQ3NjAyMjcxMTIyNTE1YzZlMzgwYmIwNWFkNjU0NGFlMzNjNTk

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -121,5 +121,5 @@ Congratulations! Your pull request has been successfully merged! 👏
 Thank you for reading through our contributing guide to ensure your
 contributions are high quality and easy for our community to review and accept. 🤝
 
-Please don't hesitate to reach out to us on [Slack](kanisterio.slack.com) if you
+Please don't hesitate to reach out to us on [Slack](https://kanisterio.slack.com) if you
```
Comment (ihcsim) [inline CONTRIBUTING.md:124]: Same as above, please use the invite link.

File: CONTRIBUTING.md (hunk 1 above)
Comment (pavannd1) [reply CONTRIBUTING.md:11]: @ihcsim Does that invite expire?

Comment (Sagar2366) [reply CONTRIBUTING.md:11]: +1

Comment (ihcsim) [reply CONTRIBUTING.md:11]: No, it's the same from the previous README, before it was revamped.

Comment (pavannd1) [reply CONTRIBUTING.md:11]: @Sagar2366 Please update the URL to the one provided by @ihcsim 👍🏼

Comment (Sagar2366) [reply CONTRIBUTING.md]: Updated to share slack invitation link.

**Pull request merged**